dgf/templates/dgf/includes/friend_detail/00_friend_header.html

```diff
@@ -1,79 +1,93 @@
 {% load static i18n %}
 
-<div id="friend-header">
-    <h1>
-        <div>
-            {{friend.full_name}}
+{% with friend.sponsors.all as all_sponsors %}
 
-            {% if friend.sponsor_logo %}
-                {% if friend.sponsor_link %}
-                    <a href="{{friend.sponsor_link}}" target="_blank">
-                        <img src="/media/{{friend.sponsor_logo}}" class="avatar-img" width="50px" height="50px"/>
-                    </a>
-                {% else %}
-                        <img src="/media/{{friend.sponsor_logo}}" class="avatar-img" width="50px" height="50px"/>
+    <div id="friend-header">
+        <div id="first-headline">
+            <div>
+                <h1>
+                    {{ friend.full_name }}
+                </h1>
+
+                {% if friend.sponsors.count %}
```

Comment:
du brauchst `count` nicht, bei python alles was `None`, 0 oder length = 0 ist immer False und alles andere True

Reply:
also `if friend.sponsors` reicht

Reply:
Ja verstehe ich. Funktioniert aber nicht so wie gewünscht. Ohne `.count` wird beispielweise der Text "Sponsored by" angezeigt obwohl man keinen Sponsor hinterlegt hat. Liegt es vielleicht daran, dass friend.sponsors hier ein queryset sind, welches existiert aber ohne Inhalt?

Reply:
In 01_friend_simple_fields.html wird es für die favorite_courses analog mit `{% if friend.favorite_courses.count %}` gemacht